docs/reference/tab-widgets/inference-api/infer-api-reindex-widget.asciidoc

```diff
@@ -27,10 +27,16 @@
     </button>
     <button role="tab"
             aria-selected="false"
-            aria-controls="infer-api-reindex-azure-openai-tab"
-            id="infer-api-reindex-azure-openai">
+            aria-controls="infer-api-reindex-azure-ai-studio-tab"
+            id="infer-api-reindex-azure-ai-studio">
       Azure AI Studio
     </button>
+    <button role="tab"
+            aria-selected="false"
+            aria-controls="infer-api-reindex-mistral"
```

Comment:
I think that will fix the missing Mistral content for the reindex step.
```suggestion
            aria-controls="infer-api-reindex-mistral-tab"
```

Reply:
OK - I _think_ this is the last of it... 😮‍💨  thank you for all your reviews on this!